### PR TITLE
Simplify guidance in GUI, extend the help file

### PR DESCRIPTION
**Pull request by Kucharssim**
The GUI now shows only this:

Screenshot 2021-11-02 at 12 42 24

and the help file has been slightly updated.

### CODE REVIEW

**Comment by AlexanderLyNL** [comment]
Yeah very good. What about this very minor adaptation?

Place each restriction on a new line, e.g., factorLow == factorHigh, where 'factor' is the factor name and 'Low'/'High' are the factor level names. Click the information icon for more examples.

**Comment by Kucharssim** [comment]
Ok, sounds good!

File: inst/qml/common/OrderRestrictions.qml [hunk 1]
```diff
@@ -30,13 +30,7 @@ Section
...
 	Text
 	{
-		text: qsTr("Place each restriction on a new line. For example:\n\nfactorLow == factorMed\nfactorMed < factorHigh\n\nwhere factor is the factor name and Low/Med/High are the factor level names.")
-	}
-
-	Text
-	{
-		text: qsTr("In case that the intercept is included,\nthe reference level of the first factor in the model is replaced by the .Intercept. keyword.\nThe corresponding syntax for the above equality restriction would be:\n\n.Intercept. == .Intercept. + factorMed\nfactorMed < factorHigh")
-		visible: type !== "RM-Anova"
+		text: qsTr("Enter each restriction on a new line, e.g., factorLow == factorHigh, \nwhere 'factor' is the factor name and 'Low'/'High' are the factor level names. \nClick the information icon for more examples.")
```
Comment (AlexanderLyNL) [inline inst/qml/common/OrderRestrictions.qml:33]: Is it necessary to add line breaks in manually? I just wonder how it behaves while scaling. @amirmasoudabdol any suggestions?

Comment (Kucharssim) [reply inst/qml/common/OrderRestrictions.qml:33]: Yeah unfortunately, this is what I mentioned during the meeting. The text can flow outside of the width of the panel currently. I am happy to hear alternative solutions.

Comment (amirmasoudabdol) [reply inst/qml/common/OrderRestrictions.qml:33]: Have you tired anchoring to left and right of the `parent`? I am not sure how `Section` behaves but this should give the `TextInput` some boundary, and then you can control the wrapping behavior using `Text::wrapMode`, which might not even be necessary.

PS. If this doesn't work, I can look into it after fixing JAGS.

**Comment by AlexanderLyNL** [comment]
Let's merge this for the moment then.

**Review by AlexanderLyNL — APPROVED** [review]

**Pull request merged**